modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/discovery/impl/DiscoveryServiceImpl.java

```diff
@@ -48,9 +48,9 @@
 public class DiscoveryServiceImpl implements DiscoveryService {
 
     public static final String FEDERATED = "Federated";
```

Comment:
What is the use of making constant FEDERATED public? Could not see any usage of this outside the class.